sr_robot_launch/scripts/teach_mode_node

```diff
@@ -35,8 +35,16 @@ from sr_robot_msgs.srv import (ChangeControlType, RobotTeachMode,
 
 
 class TeachModeManager:
+    """
+    A class to manage change in control of the robot using the ControllerHelper class.
+    It advertises a service to change the control type of the robot.
+    """
 
     def __init__(self):
```

Comment:
I think this should be type hinted as well

Reply:
fixed in https://github.com/shadow-robot/sr_interface/pull/821/commits/805b512af6ba6923a3a0838db99a20a5fbcdeaf2

Reply:
union imported in https://github.com/shadow-robot/sr_interface/pull/821/commits/4111870900b354fa9de234f6f4b04b4714441783